{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/urls.py

```diff
@@ -27,6 +28,7 @@
 urlpatterns = auth_urlpatterns + [
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/', include('{{cookiecutter.repo_name}}.apps.api.urls', namespace='api')),
+    url(r'^api-docs/', get_swagger_view(title='{{cookiecutter.repo_name}} API')),
```

Comment:
So are we ok with having this visible by default even in production for new services?  We could block the URL at the firewall if we want to, but I doubt if we'd remember to do that consistently.

Reply:
Based on conversations I have been a part of, it sounds like we are comfortable with having API docs visible by default.

Reply:
Agreed.  Hopefully we can be even more comfortable with new projects started from the cookie-cutter, since they will be starting with this from the beginning.